Presentation.Web/app/shared/helpText/helpTextModal.view.html

```diff
@@ -5,6 +5,7 @@ <h3 class="modal-title">{{title}}</h3>
     <div class="modal-body" style="word-wrap: break-word;" data-ng-bind-html="description">
     </div>
     <div class="modal-footer">
+        <button type="button" class="btn btn-success" ng-click="navigateToHelpTextEdit()">Rediger</button>
```

Comment:
missing ng-if="canEditHelpTexts" ... which is set in the controller by calling the service, which checks if the current user is global admin